package.json

```diff
@@ -6,21 +6,23 @@
   "scripts": {
     "start": "NODE_ENV=development webpack-dev-server --host=0.0.0.0 --port=9020 --inline --hot",
     "clean": "rimraf dist && rimraf gh-pages",
+    "lint": "standard",
```

Comment:
`standard` with lint every single js file we have in the project, we don't need to lint the `dist` directory for example, also can you add prebuild step that does the linting ?